modules/accredited_representative_portal/config/routes.rb

```diff
@@ -10,5 +10,6 @@
     resources :power_of_attorney_requests, only: %i[index show] do
       resource :decision, only: :create, controller: 'power_of_attorney_request_decisions'
     end
+    resources :intent_to_file, only: %i[show create]
```

Comment:
We might want to figure out a more conventional way for how this resource is represented in the URL path, but we can delay that choice until later.